SteamBot/Bot.cs

```diff
@@ -336,6 +364,24 @@ public void HandleBotCommand(string command)
             }
         }
 
+        public void HandleInput(string input)
+        {
+            consoleInput = input;
+        }
+
+        public string WaitForInput()
+        {
+            consoleInput = null;
+            while (true)
+            {
+                if (consoleInput != null)
+                {
+                    return consoleInput;
+                }
+                Thread.Sleep(5);
+            }
+        }
```

Comment:
Feel free to make a pull request later if you feel it's necessary, but I think it's important that we get mobile auth working with SteamBot ASAP first.